### PR TITLE
Issue #SB-19637 feat: filter on content mimetype

### PR DESCRIPTION
**Pull request by AniketSaki**
Filtering the content consumption report on mimetype as provided on https://project-sunbird.atlassian.net/browse/SB-19637
The mimetype filters are stored in a config file on blob store as the filters may change.
Added functionality in fetching content plays to filter on mimetype if the query date range is after 2 July (when content mimetype was indexed in druid datasource.

### CODE REVIEW

File: python-scripts/src/main/python/dataproducts/resources/queries/content_list.py [hunk 1]
```diff
@@ -14,8 +14,8 @@ def init():
                 },
                 {
                     "type": "in",
-                    "dimension": "contentType",
-                    "values": ["Resource", "ExplanationResource", "FocusSpot", "PracticeQuestionSet", "eTextBook", "LearningOutcomeDefinition", "PracticeResource", "ExperientialResource", "SelfAssess", "CuriosityQuestionSet"]
+                    "dimension": "mimeType",
+                    "values": "mimetype_filter"
```
Comment (SanthoshVasabhaktula) [inline python-scripts/src/main/python/dataproducts/resources/queries/content_list.py:18]: Rename the current query to content_list_v1.py and create a new query

File: python-scripts/src/main/python/dataproducts/util/utils.py [hunk 2]
```diff
@@ -274,13 +275,33 @@ def get_content_model(result_loc_, druid_, date_, status_=["Live"]):
...
+            'video/3gpp': 'Video',
+            'video/avi': 'Video',
+            'video/ogg': 'Video',
+            'video/quicktime': 'Video',
+            'video/mp4': 'Video',
+            'video/mpeg': 'Video',
+            'video/webm': 'Video',
+            'video/x-youtube': 'YouTube'
+        }
+        content_model['mimeType'].replace(rename_mimetype, inplace=True)
```
Comment (SanthoshVasabhaktula) [inline python-scripts/src/main/python/dataproducts/util/utils.py:304]: move the mapping to a config object

File: python-scripts/src/main/python/dataproducts/util/utils.py [hunk 3]
```diff
@@ -547,7 +568,16 @@ def get_content_plays(result_loc_, start_date_, end_date_, druid_, config_):
...
+    query = json.loads(query)
+    if start_date_ > datetime(2020, 7, 2):
+        query['filter']['fields'].append(
+            {
+                "type": "in",
+                "dimension": "content_mimetype",
+                "values": config_['content']['mimeType']
+            },
+        )
+    response = requests.post(url, data=json.dumps(query), headers=headers)
```
Comment (SanthoshVasabhaktula) [inline python-scripts/src/main/python/dataproducts/util/utils.py:580]: this logic is not required. Let the job take in a parameter for the query version to run and the invoker of the job should know the version to use

File: python-scripts/src/main/python/dataproducts/resources/queries/content_list.py (hunk 1 above)
Comment (AniketSaki) [reply python-scripts/src/main/python/dataproducts/resources/queries/content_list.py:18]: Should I still be reading the whitelist filter from the config object or have it in the new query itself?

**Pull request merged**